# Msg that identifies the behavior that sends a desired velocity.
uint16 node_ID
PolarVelocity2D velocity
